### ISSUE TITLE
[Bug] perl code cannot be generated in 3.1 [sf#42]

### ISSUE THREAD

**Issue by DietmarSchwertberger** [DietmarSchwertberger/wxGlade#89]
**Reported by equinox4 on 2003-09-02 14:44 UTC**
I created a frame, within that I created a sizer and 
within that a label.

It generates code for Python and C++ but it does not 
for Perl.

Error:

no module named edit_sizers.perl_sizers_codegen exists.

I downloaded the wxGlade-0.3.1-setup.exe.

Anteusz

**Comment by DietmarSchwertberger** [comment]
**Commented by agriggio on 2003-09-02 22:38 UTC**
Logged In: YES 
user_id=375945

quite strangely, I wasn't able to reproduce this here... :-(
I don't know why, but anyway I just sent a note to the
mailing list, to see if someone else can confirm it. 

If this is the case, I will upload a new -setup.exe

In the meantime, if you have python + wxPython installed,
the source distribution should work withouth problems...

**Comment by DietmarSchwertberger** [comment]
**Commented by dafire on 2003-09-18 12:05 UTC**
Logged In: YES 
user_id=170482

I have the same problem.

**Comment by DietmarSchwertberger** [comment]
**Commented by agriggio on 2003-09-18 18:53 UTC**
Logged In: YES 
user_id=375945

I've uploaded a wxGlade-0.3.1-1-setup.exe
It should work (I've tested it on a &quot;clean&quot;) machine, but
please, if cou can, confirm it. I'll keep the bug open until
I receive at least one positive response

Thanks,
Alberto

**Comment by DietmarSchwertberger** [comment]
**Commented by crazyinsomniac on 2003-09-20 09:59 UTC**
Logged In: YES 
user_id=264900

I tried it out on Win2K Pro SP3,
and I have no problems.
You guys should report your OS/version/SP.

**Comment by DietmarSchwertberger** [comment]
**Commented by dafire on 2003-09-21 18:55 UTC**
Logged In: YES 
user_id=170482

i installed it on my fresh installed laptop and perl generation 
works. I'll try it during the week on the box where I had the 
problems but looks like it works now fine

**Comment by DietmarSchwertberger** [comment]
**Commented by equinox4 on 2003-09-22 09:54 UTC**
Logged In: YES 
user_id=857391

Ok.
I ran it on Winnt 4.0.


Anteusz

**Comment by DietmarSchwertberger** [comment]
**Commented by equinox4 on 2003-09-22 10:31 UTC**
Logged In: YES 
user_id=857391

Ok. I downloaded and tried the new wxGlade-0.3.1-1 
setup.exe.

It seems to work.
This problem can be closed.

BUT I found a smalll problem. Previously I generated a python 
code in glade.pl

Now I set the language to Perl, and pressed generate.
It generated a python code despite of the setting!!!!
I suspect it determined the language from glade.pl because 
when I deleted glade.pl, it correctly generated a perl code in 
the new glade.pl.

Anyway , the main problem is solved.


Anteusz
